api/src/main/scala/hmda/api/http/institutions/SubmissionPaths.scala

```diff
@@ -141,4 +141,39 @@ trait SubmissionPaths
       }
 
     }
+
+  def submissionSingleEditPath(institutionId: String) =
+    path("filings" / Segment / "submissions" / IntNumber / "edits" / Segment) { (period, submissionId, editType) =>
+      val path = s"institutions/$institutionId/filings/$period/submissions/$submissionId/edits/$editType"
+      extractExecutionContext { executor =>
+        implicit val ec: ExecutionContext = executor
```

Comment:
It looks like these lines (149, 150, and maybe 151) are duplicated in the `submissionEditsPath` endpoint. Is it possible to extract them so the methods can share common code?